app/assets/stylesheets/components/_email-link.scss

```diff
@@ -1,16 +1,19 @@
 .app-c-email-link {
   display: block;
-  font-weight: bold;
   padding: govuk-spacing(4);
   background: govuk-colour('grey-4');
 
+  .app-c-email-link__title {
+    font-weight: bold;
+  }
+
+  .app-c-email-link__icon {
+    position: relative;
+    margin-right: govuk-spacing(2);
+    top: govuk-spacing(1);
```

Comment:
I would set this to an explicit `5px`. Only because some larger `govuk-spacing` numbers have different values on mobile vs desktop which is not what we want in this case. It doesn't affect `govuk-spacing(1)` but it makes it easier for someone to simply change it to `govuk-spacing(6)`, for example, without considering that mobile vs desktop difference.

Reply:
Great shout, will amend now!

Reply:
done

Reply:
@huwd Looks like this is still `govuk-spacing(1)`?

Reply:
Ah, looks like I got the wrong end of the stick and changed the margin-right above. Have reverted that and edited like `13`